src/scripts/datamanager.ts

```diff
@@ -166,7 +166,8 @@ export class DataManager {
                 }
             }).then((rawItems) => {
                 this.allItems = rawItems
-                    .filter(i => i['Stats'] !== null)
+                    // TODO: filtering out items with no stats excludes all BLU weapons
+                    // .filter(i => i['Stats'] !== null)
```

Comment:
```suggestion
                    .filter(i => {
                        return i['Stats'] !== null
                            || (i['ClassJobCategory']?.['BLU'] === 1 && i['EquipSlotCategory']?.['MainHand'] === 1);
                    })

```

GH won't let me leave a suggestion for something that's not actually part of the PR, but you'd have to add `'ClassJobCategory'` to the columns that it fetches.

Reply:
Would that be the preferred approach?  Right now, the hard limits on item levels set in `xivconstants.ts` is preventing any of the glamour gear from getting fetched, so the filter is really only excluding the BLU weapons as far as I can see.

Reply:
I think it's the safest option, since it limits the scope of the change as much as possible.

Reply:
Ok.  I was just thinking in terms of data size of the return request, adding `ClassJobCategory` to every item is going to have a much bigger impact.